UOP1_Project/Assets/Scripts/Cutscenes/DialogueTimeline/DialogueBinder.cs

```diff
@@ -0,0 +1,11 @@
+
+namespace UOP1.Cutscenes
+{
+	/// <summary>
+	/// Inherit this class to create a target for the timelines dialogue
+	/// </summary>
+	public abstract class DialogueBinder : UnityEngine.Component
+	{
+		public abstract void SetDialog(string dialogueLineID);
```

Comment:
The idea of calling a dialogue line from a ScriptableObject using a string sounds like a dangerous idea, it's prone to mistakes and doesn't work well when you decide to move lines around. Now you need to manually go and look into clips to change the IDs.